media/html/visjsDebuggerPanel.html

```diff
@@ -187,17 +221,19 @@
 
       function openClusterStack(clusterStack) {
         clusterStack.forEach((cluster) => {
-          if (nodes.get(cluster) !== undefined && network.isCluster(cluster)) {
+          if (nodes.get(cluster) && network.isCluster(cluster)) {
```

Comment:
```suggestion
          if (network.isCluster(cluster)) {
```
`nodes.get(cluster)` always returns null because the clusters are never added to the `nodes`. Checking for `nodes.get(cluster)` always returns false and thus no cluster is every opened here. `nodes.get(cluster) !== undefined` worked  because `null !== undefined` but it was wrong. (The button doesn't work currently because of this change)